<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Availability_Window</class>
 <widget class="QMainWindow" name="Availability_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Availabilty_Window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>591</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>470</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Filter Workers </string>
    </property>
    <widget class="QPushButton" name="bartenderBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Bartenders</string>
     </property>
    </widget>
    <widget class="QPushButton" name="serverBtn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Servers</string>
     </property>
    </widget>
    <widget class="QPushButton" name="expoBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Expo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hostBtn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Host</string>
     </property>
    </widget>
    <widget class="QPushButton" name="busBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Bus</string>
     </property>
    </widget>
    <widget class="QPushButton" name="managerBtn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Manager</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Submit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>510</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit AVailability </string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>170</y>
      <width>201</width>
      <height>301</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AM Shift </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>440</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PM Shift</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RemoveAMBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>600</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove AM Employee</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RemovePMBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>600</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove PM Employee</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>885</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionE_xit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>Availability_Window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calendarWidget</sender>
   <signal>selectionChanged()</signal>
   <receiver>Availability_Window</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bartenderBtn</sender>
   <signal>clicked()</signal>
   <receiver>Availability_Window</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>busBtn</sender>
   <signal>clicked()</signal>
   <receiver>Availability_Window</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hostBtn</sender>
   <signal>clicked()</signal>
   <receiver>Availability_Window</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>expoBtn</sender>
   <signal>clicked()</signal>
   <receiver>Availability_Window</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serverBtn</sender>
   <signal>clicked()</signal>
   <receiver>Availability_Window</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>managerBtn</sender>
   <signal>clicked()</signal>
   <receiver>Availability_Window</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RemovePMBtn</sender>
   <signal>clicked()</signal>
   <receiver>Availability_Window</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>648</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RemoveAMBtn</sender>
   <signal>clicked()</signal>
   <receiver>Availability_Window</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>648</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Submit</sender>
   <signal>clicked()</signal>
   <receiver>Availability_Window</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
